addons/web_editor/static/src/js/editor/toolbar.js

```diff
@@ -2,6 +2,7 @@ odoo.define('web_editor.toolbar', function (require) {
 'use strict';
 
 var Widget = require('web.Widget');
+var config = require('web.config');
```

Comment:
I'm gonna seize my opportunity to scold @dmo-odoo here for not writing a commit message. A chance like that is once in a lifetime. So David, consider yourself scolded. :partying_face:

Reply:
I have done so, good sir. 🫥